internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -338,10 +338,25 @@ var rules = []string{
 	"dont_measure fsmagic=0x43415d53", // SMACK_MAGIC
 	"dont_measure fsmagic=0x27e0eb",   // CGROUP_SUPER_MAGIC
 	"dont_measure fsmagic=0x63677270", // CGROUP2_SUPER_MAGIC
-	"dont_measure fsmagic=0x6e736673", // NSFS_MAGIC
+	"dont_measure fsmagic=0x6e736673", // NFS_MAGIC
 	"dont_measure fsmagic=0xde5e81e4", // EFIVARFS_MAGIC
 	"dont_measure fsmagic=0x58465342", // XFS_MAGIC
 	"dont_measure fsmagic=0x794c7630", // OVERLAYFS_SUPER_MAGIC
+	"dont_measure fsmagic=0x858458f6", // BTRFS_SUPER_MAGIC
```

Comment:
any links to where these magic numbers are from, the kernel docs doesn't seem to have ext4 ones, at least the one in the comment

Reply:
https://man7.org/linux/man-pages/man2/statfs.2.html

Reply:
ha... this one is wrong, I will fix it up, thanks for noticing that https://github.com/torvalds/linux/blob/master/include/uapi/linux/magic.h

Reply:
ok, now I'm more confused... I'll dig into that

Reply:
https://github.com/torvalds/linux/blob/v5.15/include/uapi/linux/magic.h we'd probably need this, since our kernel is 5.15

Reply:
I guess magics shouldn't change, they're derived from the filesystem superblock. should be better now